### PR TITLE
Move tests away from Travis

### PR DESCRIPTION
**Pull request by chessbr**
Keep Browser tests still on Travis for now

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -42,38 +44,9 @@ jobs:
...
-script:
-  - tox -vv
-after_success:
-  - coveralls
-  - codecov
-notifications:
-  slack:
-    secure: mkYYYt9ZQc6vR1aJltnEF1DzZ5xsNmllNcdK8fCwjeK5ZOnB1atujZ08tYxx/t57gaxhMWyJHvKdD0gvyQS6R/pAO2epXY9NqOhPVDEhqtXBpd7wUcEMTQqjSy68FSTjO94O8suC18xB55Cq+KSxm8jJt6zSFsjoffhz+4QIA67rkiNPLmAw2AHYymvVaidWMDe5UUGD7qONmdFI5wx4wOE9veboWDQ49oygiq+c4NFC/d4C108jT7l9ZTko/JF7Bim3VQI0LprGuIH2tZ6TI/3LWEhQX6XXuGr6eXuTF3nQp470yRXRs50Gh+CSY++ZUgFdRfOlg0edHt9m2Lk8j2jfeLgXYWYy1QSCnJDPRzzVIkEP4xaZCcIK7pcckV0Ytn7BP3Y09MDzSvt3+0Cm7ou16shiyy5NqW5DonarN21Mlz9/auZPawsXJqYZVDIZQiM7GLJnpg9wBfgKlPxUZKwi5ttsvrS7iPNK1sSLmICFqukrQlMQnpY5h45uGXsqL7fI7z30Ftld6qIGGVFdFjh3K+4dVXV5XKHnynaR518JvugxLwrNcIWwpyB2I3Ak7AsMzd3IWnY4yrB6pSbVdG0W3zZYXR6GUsYmxICKv5pUVnRKtp2tgOKA3EwgnVjSkJahGPp5bc3xJN8mNOaR33f3kieM5jJcNteEWueX+eA=
+        - pip install -r requirements-tests.txt
+        - python setup.py build_resources --ci
```
Comment (ruohola) [inline .travis.yml:48]: What does this `--ci` flag do=

File: setup.cfg [hunk 2]
```diff
@@ -35,3 +35,26 @@ requirements-doc =
...
+    isort==5.6.4
+    flake8==3.8.4
+    prequ
+
+requirements-tests =
+    -e .
+    codecov
+    cryptography==3.3.1  # Last known version that builds with Travis
+    beautifulsoup4==4.5.3
+    html5lib==0.999999999
```
Comment (ruohola) [inline setup.cfg:51]: :DDD wtf is this versioning

Comment (chessbr) [reply setup.cfg:51]: copied from tox, but that version has a really nice history behind it

File: .travis.yml (hunk 1 above)
Comment (chessbr) [reply .travis.yml:48]: image

Comment (chessbr) [reply .travis.yml:48]: image

**Pull request merged**